tests/simulation_test.py

```diff
@@ -0,0 +1,93 @@
+import unittest
+
+import numpy
+
+from worldengine.simulations.hydrology import WatermapSimulation
+from worldengine.model.world import World, Size, GenerationParameters
+
+class TestSimulation(unittest.TestCase):
+    
+
+    # The hydrology simulation indirectly calls the global rng.
+    # We want different implementations of _watermap 
+    # and internally called functions (especially random_land)
+    # to show the same rng behaviour and not contamine the state of the global rng
+    # should anyone else happen to rely on it.
+    # This does only test that watermap leaves the rng in the state that the 
+    # original implementation would have left it in and that a very small sample 
+    # of results is the same as the original results.
+    # It does not test if the implementation is actually correct. 
+    # In fact it is hard to tell how "correctness" of a Monte Carlo process should
+    # be judged.
+    # Should it ever be decided that a "correct" implementation needs fewer or 
+    # more calls to the rng, relative to the n parameter,
+    # of course compatibility will be broken. 
+    # Implicitly this is tested by the blessed images but it might be useful to
+    # have a finer grained picture.
+
+    def test_watermap_rng_stabilty(self):
+        
```

Comment:
nitpick: I think we normally do not put a white line after the header of the function